credoai/metrics/metrics.py

```diff
@@ -0,0 +1,140 @@
+from absl import logging
```

Comment:
This is the main new functionality. Most of the other changes are dealing with this change.